pkgs/development/libraries/freetype/default.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-R2byAVfMTPDNKS+Av5F/ktHEObJDrDAY3r9rkUDEGn8=";
   };
 
-  propagatedBuildInputs = [ zlib bzip2 libpng ]; # needed when linking against freetype
+  propagatedBuildInputs = [ zlib bzip2 brotli libpng ]; # needed when linking against freetype
```

Comment:
it appears to not be necessary to propagated any of these anymore(at least nothing up to blender failed). so please move brotli to buildInputs
```
/nix/store/206av96grdv4n0afilw9y3ws4vdjlrjr-blender-3.3.1
├───bin/blender: …ATH=${PYTHONPATH/':''/nix/store/2p2ag2y9rz30pnjw68n2kbxgrvx8z0ii-python3.10-brotlicffi-1.0.9.2/l…
│   → /nix/store/2p2ag2y9rz30pnjw68n2kbxgrvx8z0ii-python3.10-brotlicffi-1.0.9.2
│   └───lib/python3.10/site-packages/brotlicffi/_brotlicffi.abi3.so: …LIBC_2.4.GLIBC_2.2.5./nix/store/kmigdljiv59kh22jr>
│       → /nix/store/kmigdljiv59kh22jrx5hp0savn2xx3fc-brotli-1.0.9-lib
├───bin/.blender-wrapped: …bpng-apng-1.6.37/lib:/nix/store/7wwmllxrf5ps40m8b44dgazn4bai1yb3-freetype-2.12.1/lib:/nix/st>
│   → /nix/store/7wwmllxrf5ps40m8b44dgazn4bai1yb3-freetype-2.12.1
│   └───lib/libfreetype.la: …ib/libpng16.la -lz -L/nix/store/kmigdljiv59kh22jrx5hp0savn2xx3fc-brotli-1.0.9-lib/lib -lbr>
│       lib/libfreetype.so.6.18.3: …yhsx-zlib-1.2.13/lib:/nix/store/kmigdljiv59kh22jrx5hp0savn2xx3fc-brotli-1.0.9-lib/l>
```

Reply:
well idk, the freetype2.pc contains `Requires.private: zlib, bzip2, libpng, libbrotlidec` which are required to static packages?

Reply:
i guess lets be safe and propagate it